CHANGELOG.md

```diff
@@ -4,7 +4,13 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/) 
-and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [0.3.1]
+RAiDER package was refactored to use a __main__ file to allow calls to different functionality.
+The default is `calcDelays` which maintains the original functionality of calling `raider.py`.
+`raider.py ++process downloadGNSS ...` can now perform the functionality of `raiderDownloadGNSS.py ...`
+`raider.py ++calcDelaysARIA GUNW` is enabled but not fully functional
```

Comment:
There might be many differnt ARIA products. Suggest to call it it "CalcDelaysGUNW" would also make GUNW more descriptiove you mean GUNW filename.

Reply:
Done